scripts/quarantine.yaml

```diff
@@ -8,7 +8,6 @@
 #    - None
 
 - scenarios:
-    - sample.tfm.hello_world.lvl2
+    - None
```

Comment:
Same problematic commit message:

- strange and unexpected leading `-`
- missing explanation: **why** is the change being made, not **what** is the change that is being made

Reply:
I removed the - and update the commit message. I hope that it is better now.